src/lm.jl

```diff
@@ -185,6 +185,31 @@ function adjr2(obj::LinearModel)
     1 - (1 - r²(obj))*(n-1)/(n-p)
 end
 
+struct FStatistic
+    nobs::Float64
+    ndims::Int64
+    f::Float64
+    fprob::Float64
+end
+
+function StatsModels.fstatistic(obj::LinearModel)
+    rss = deviance(obj)
+    tss = nulldeviance(obj)
+
+    n = nobs(obj)
+    p = dof(obj) - 2 # -2 for dispersion parameter and intercept
+    fstat = ((tss - rss) / p) / (rss / (n - p - 1))
+    fdist = FDist(n, p)
+
+    FStatistic(n, p, fstat, ccdf.(fdist, abs(fstat)))
```

Comment:
why the broadcast `.` here?

Reply:
I'd like to ask yesterday's me that question :- )